upload.py

```diff
@@ -992,10 +998,13 @@ def draw(self, context):
         if self.reupload:
             utils.label_multiline(
                 layout,
-                text="To update only metadata of the model, keep checkboxes unchecked",
+                text=(
+                    "Select the components of the asset you wish to update through reupload. "
+                    "For editing metadata and thumbnails, you can also use the web interface linked above."
```

Comment:
Je skutečně odkaz "above", nebo je to ten button "Edit"? V takovém případě bych to možná popsal konkrétně s tím co je napsané na tom buttonu.